### DISCUSSION TITLE
Connect to Asterisk CLI

### DISCUSSION THREAD

**Discussion by bkbilly** [TECH7Fox/asterisk-hass-addons#335] [Q&A] [answered]
Is it possible to connect to the Asterisk CLI when installed on a Raspberry as a Home Assistant Addon?
On my setup this is done with the command `asterisk -rvvv`.

**Comment by felipecrs** [comment, marked as answer]
https://tech7fox.github.io/sip-hass-docs/docs/add-on/troubleshooting#accessing-asterisk-cli
